pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md

```diff
@@ -560,4 +561,47 @@ Each basic element can have specific style rules defined for it, controlling how
 }
 ```
 
-For a list of Rich Messaging capabilities for each channel, browse or search the table on the [Knowledge Center](https://knowledge.liveperson.com/messaging-channels-messaging-channels-capabilities-comparison.html).
\ No newline at end of file
+### Element Accessibility Attributes
+
+Each basic element can have specific accessibility attributes defined to them. 
```

Comment:
Please add another line if you want two paragraphs, or remove the new line to make this one paragraph :)